src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -289,7 +289,7 @@ def _init_existing_flow(flow_name, entry=None, function=None, prompt_params: dic
     tools.generate_to_file(meta_dir / "flow.tools.json")
     # Create flow.dag.yaml
     FlowDAGGenerator(tool_py, function, function_obj, prompt_params).generate_to_file("flow.dag.yaml")
-    copy_extra_files(flow_path=flow_path, extra_files=[".gitignore"])
+    copy_extra_files(flow_path=flow_path, extra_files=["requirements.txt", ".gitignore"])
```

Comment:
do we handle case that user already have a requrements.txt?

Reply:
Currently, it will overwrite existing requriements.txt. 
It may be necessary to deal with it here, which of the following is more appropriate:
1. Add promptflow to the existing requirements.txt
2. Print warning log to alert the user to add promptflow in requirements.txt

Reply:
offline sync, if requirments.txt exist, skip it.